src/views/forms/NewMarketForm/v7/NewMarketPreviewStep.tsx

```diff
@@ -0,0 +1,216 @@
+import { FormEvent, useMemo, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { AlertType } from '@/constants/alerts';
+import { ButtonAction, ButtonType } from '@/constants/buttons';
+import { DialogTypes } from '@/constants/dialogs';
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+import { formMixins } from '@/styles/formMixins';
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { AlertMessage } from '@/components/AlertMessage';
+import { AssetIcon } from '@/components/AssetIcon';
+import { Button } from '@/components/Button';
+import { Details } from '@/components/Details';
+import { Icon, IconName } from '@/components/Icon';
+import { Output, OutputType } from '@/components/Output';
+
+import { useAppDispatch } from '@/state/appTypes';
+import { openDialog } from '@/state/dialogs';
+
+import { log } from '@/lib/telemetry';
+
+type NewMarketPreviewStepProps = {
+  ticker: string;
+  onBack: () => void;
+  onSuccess: (ticker: string) => void;
+  receiptItems: Parameters<typeof Details>[0]['items'];
```

Comment:
I think there's a PropType type somewhere that gives you the props which might be easier to read here